CONTRIBUTING.md

```diff
@@ -85,19 +90,30 @@ copyright holders.
 
 ## Sign your work
 
-This project tracks patch provenance and licensing using a modified Developer
-Certificate of Origin (DCO; from [OSDL][DCO]) and Signed-off-by tags initially
-developed by the Linux kernel project.
+This project tracks patch provenance and licensing using the Developer
+Certificate of Origin 1.1 (DCO) from [developercertificate.org][DCO] and
+Signed-off-by tags initially developed by the Linux kernel project.
 
-```
+```text
+Developer Certificate of Origin
+Version 1.1
+
+Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
+1 Letterman Drive
+Suite D4700
+San Francisco, CA, 94129
+
+Everyone is permitted to copy and distribute verbatim copies of this
+license document, but changing it is not allowed.
 
-pyLife Developer's Certificate of Origin.  Version 1.1
+
+Developer's Certificate of Origin 1.1
 
 By making a contribution to this project, I certify that:
 
 (a) The contribution was created in whole or in part by me and I
-    have the right to submit it under the "Apache License, Version 2.0"
-    ("Apache-2.0"); or
+    have the right to submit it under the open source license
+    indicated in the file; or
```

Comment:
Shoudn't we directly refer to the license as being Apache 2 instead of "open source licensed indicated in the file"

Reply:
That's what we had in our modified version. The benefits of using the standard DCO as everybody else (e.g., Linux kernel) far outweigh any custom version IMO. The Apache-2.0 requirement is still in the Contributing file. As we also require Copyright / License headers, it's equivalent (with one indirection... )

Reply:
Ok. I will merge it then.

Reply:
I just saw, that the DCO app is already activated. Incidentally it also links to the standard DCO 1.1: https://probot.github.io/apps/dco/